src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
Here you can use destructuring to improve code readability.

Reply:
In this case i use almost all fields of this object. so i deliberately don't use destructuring here.
And this is not the reason to reject. What's wrong with you, man?

Reply:
No promble, just previously mentors rejected my solution in similar situation.

Reply:
It's just recommendation, but not a rule